test/zmumu/tp_from_aod_MC.py

```diff
@@ -489,8 +491,8 @@
         arbitration   = "OneProbe",
         variables = cms.PSet(
             KinematicVariables, 
-            bx = cms.string("bx"),
```

Comment:
BX won't work anymore as stage2 trigger object do not store BX information
Quality is easier to fix but I would let experts fix the two talltoghether